components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -313,6 +313,7 @@ class RewardsServiceImpl : public RewardsService,
                                  bool success);
   void OnTipPublisherInfoSaved(ledger::Result result,
                                    ledger::PublisherInfoPtr info);
+  void OnDoDirectTip(int result);
```

Comment:
Should this be `ledger::Result` instead of a raw int?

Reply:
Sicne this callback is passed as a mojo callback, and the mojo interface doesn't know about `ledger::Result`, it needs to be an `int`. If any code is added in this function it should cast to `ledger::Result`, but I can't leave that cast in the function without any code that uses the actual result because I'll get an unused variable warning.

Reply:
Makes sense.